packages/native-core/src/layout_attributes.rs

```diff
@@ -19,573 +19,400 @@
-
-        "border"
-        | "border-bottom"
-        | "border-bottom-color"
-        | "border-bottom-left-radius"
-        | "border-bottom-right-radius"
-        | "border-bottom-style"
-        | "border-bottom-width"
-        | "border-collapse"
-        | "border-color"
-        | "border-image"
-        | "border-image-outset"
-        | "border-image-repeat"
-        | "border-image-slice"
-        | "border-image-source"
-        | "border-image-width"
-        | "border-left"
-        | "border-left-color"
-        | "border-left-style"
-        | "border-left-width"
-        | "border-radius"
-        | "border-right"
-        | "border-right-color"
-        | "border-right-style"
-        | "border-right-width"
-        | "border-spacing"
-        | "border-style"
-        | "border-top"
-        | "border-top-color"
-        | "border-top-left-radius"
-        | "border-top-right-radius"
-        | "border-top-style"
-        | "border-top-width"
-        | "border-width" => apply_border(name, value, style),
-
-        "bottom" => {}
-        "box-shadow" => {}
-        "box-sizing" => {}
-        "caption-side" => {}
-        "clear" => {}
-        "clip" => {}
-
-        "column-count"
-        | "column-fill"
-        | "column-gap"
-        | "column-rule"
-        | "column-rule-color"
-        | "column-rule-style"
-        | "column-rule-width"
-        | "column-span"
-        // add column-width
-        | "column-width" => apply_column(name, value, style),
-
-        "columns" => {}
-
-        "content" => {}
-        "counter-increment" => {}
-        "counter-reset" => {}
-
-        "cursor" => {}
-        "direction" => {}
-
-        "display" => apply_display(name, value, style),
-
-        "empty-cells" => {}
-
-        "flex"
-        | "flex-basis"
-        | "flex-direction"
-        | "flex-flow"
-        | "flex-grow"
-        | "flex-shrink"
-        | "flex-wrap" => apply_flex(name, value, style),
-
-        "float" => {},
-
-        "font-style"
-        | "font-variant"
-        | "font-weight"
-        | "font-size"
-        | "line-height"
-        | "font-family" => apply_font(name, value, style),
-
-        "height" => {
-            if let Some(v) = parse_value(value){
-                style.size.height = v;
+    if let Ok(property) =
+        Property::parse_string(PropertyId::from(name), value, ParserOptions::default())
+    {
+        match property {
+            Property::Display(display) => match display {
+                lightningcss::properties::display::Display::Keyword(_) => todo!(),
+                lightningcss::properties::display::Display::Pair(pair) => {
+                    match pair.outside {
+                        lightningcss::properties::display::DisplayOutside::Block => {
+                            style.display = Display::None
+                        }
+                        lightningcss::properties::display::DisplayOutside::Inline => todo!(),
```

Comment:
I feel like these `todo()`'s shouldn't be included, as they will cause the code to panic if the relevant style is specified. I think it would be better to silently ignore unsupported styles, or else surface a warning/error in a more graceful way